### PR TITLE
CWI 638: Update 3 free mural message

### PR DESCRIPTION
**Pull request by KatieHendricksen**
image (149)

### CODE REVIEW

File: packages/mural-picker/src/components/mural-create/index.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@ import { getCommonTrackingProperties } from '../../common/tracking-properties';
 import { CardListSection } from '../card-list/card-list-section';
 import { ErrorHandler } from '../types';
 
+import { getApiError } from '../../../../mural-client/src';
```
Comment (KavenTheriault) [inline packages/mural-picker/src/components/mural-create/index.tsx:19]: I don't know why I missed that, but I think this line is the issue

Should be:
```
import {
  getApiError,
} from '@muraldevkit/mural-integrations-mural-client';
```

Comment (KatieHendricksen) [reply packages/mural-picker/src/components/mural-create/index.tsx]: Oh!! Thank you!

File: packages/mural-picker/src/components/mural-create/index.tsx [hunk 2]
```diff
@@ -16,6 +16,7 @@ import { getCommonTrackingProperties } from '../../common/tracking-properties';
 import { CardListSection } from '../card-list/card-list-section';
 import { ErrorHandler } from '../types';
 
+import { getApiError } from '@muraldevkit/mural-integrations-mural-client';
```
Comment (KavenTheriault) [inline packages/mural-picker/src/components/mural-create/index.tsx:19]: Can you merge it with the import at line 4 ?

Comment (KatieHendricksen) [reply packages/mural-picker/src/components/mural-create/index.tsx]: good idea!

**Review by KavenTheriault — APPROVED** [review]
LGTM

**Pull request merged**